### PR TITLE
make x-tag work for CommonJS (for Browserify)

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

File: src/core.js [hunk 1]
```diff
@@ -841,8 +841,9 @@
 
   };
 
-  win.xtag = xtag;
-  if (typeof define == 'function' && define.amd) define(xtag);
+  if (typeof define === 'function' && define.amd) define(xtag);
```
Comment (cvan) [inline src/core.js]: I would do braces and separate lines, but you seem to not like those

Comment (csuwildcat) [inline src/core.js]: Why pull the window object addition?

Comment (cvan) [inline src/core.js]: because if I'm doing CommonJS w/ Browserify, I don't want you polluting a global, brah

Comment (cvan) [inline src/core.js]: and, I'm just [moving it](https://github.com/x-tag/core/pull/126/files#diff-7eb52b366866677666470e019283c8eaR846)

Comment (csuwildcat) [inline src/core.js]: That works :)

**Pull request merged**